package.json

```diff
@@ -38,7 +38,7 @@
     "tmp": "^0.2.1"
   },
   "devDependencies": {
-    "@percy/cli": "^1.25.0",
+    "@percy/cli": "^1.27.0-beta.0",
```

Comment:
Nice Catch!
Ideally this should not be removed, since this is dependency.
Hence, updating this to `1.27.0-beta.0` as well.

Reply:
I was talking about `sdk-utils` actually because now `percy/cli` will download `webdriver-utils` and `webdriver-utils` has `sdk-utils` so it will get downloaded, `transitive dependency`.

Reply:
Both are at same version level now, so should be fine imo.